pkgs/applications/version-management/yadm/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchurl, git, bash }:
+
+let version = "1.04"; in
+let link = "https://raw.githubusercontent.com/TheLocehiliosan/yadm/${version}"; in
+stdenv.mkDerivation {
+  name = "yadm-${version}";
+  isLibrary = false;
+  isExecutable = true;
+
+  exe = fetchurl {
+    url = "${link}/yadm";
+    sha256 = "c2a7802e45570d5123f9e5760f6f92f1205f340ce155b47b065e1a1844145067";
+  };
+
+  man = fetchurl {
+    url = "${link}/yadm.1";
+    sha256 = "868755b19b9115cceb78202704a83ee204c2921646dd7814f8c25dd237ce09b2";
+  };
+
+  builder = ./builder.sh;
```

Comment:
I think it would probably be easiest to just set "buildCommand" to the contents of ./builder.sh